packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -386,7 +388,10 @@ export default class ComboBox extends React.Component {
               />
             </ListBox.Field>
             {isOpen && (
-              <ListBox.Menu aria-label={ariaLabel} id={id}>
+              <ListBox.Menu
+                aria-label={ariaLabel}
+                id={id}
+                {...getMenuProps({ 'aria-label': ariaLabel })}>
```

Comment:
If we pass in `aria-label` to `getMenuProps` do they return it so we don't have to set `aria-label` above?